ci/kokoro/docker/build-in-docker-cmake.sh

```diff
@@ -98,7 +98,8 @@ if [[ ${RUN_INTEGRATION_TESTS} == "yes" ]]; then
   # Run the integration tests too.
   env -C "${BINARY_DIR}" ctest \
       -L integration-tests \
-      --output-on-failure
+      --output-on-failure \
+      -j 4
```

Comment:
Might as well make this `"$(nproc)"` to be consistent?

Reply:
Well, actually we can run test in parallel even on single core machine because I believe they are in separate processes and most of the time is spent waiting for network io.

Reply:
There is a mismatch between `-j` (run `N` jobs in parallel) and `nproc` (the number of processing units available).  In general I think you want to scale `nproc` by some "job load factor" to get `N`.  If the jobs are CPU bound, the factor is 1, but for I/O bound jobs it could be significantly higher.